### ISSUE TITLE
Caret jumps after adding new JSDoc block

### ISSUE THREAD

**Issue by mrapczynski** [joelday/vscode-docthis#70]
As the title explains, the caret jumps up in a funny way after hitting Ctrl+Alt+D twice. Currently using VS Code `1.8.1` with Document This `0.3.6`

Here is a GIF to demonstrate:

![document-this-caret-jump](https://cloud.githubusercontent.com/assets/258614/21817749/0db8ae52-d71a-11e6-8124-f7e2f668e354.gif)

**Comment by joelday** [comment]
Yikes. Looking at this now.

**Comment by joelday** [comment]
@mrapczynski Just published 0.3.7 which should address this. Let me know if you're still having issues.

**Comment by mrapczynski** [comment]
@joelday I tried it now just now, and the issue still seems to be there. Another GIF showing that I have 0.3.9 of the extension installed.

![caret-jump-v2](https://cloud.githubusercontent.com/assets/258614/21855610/e0eff28a-d7d3-11e6-8126-bc14162325b5.gif)

**Comment by joelday** [comment]
@mrapczynski Strange. I wasn't able to repro it. Can you paste a list of your installed extensions and your settings? Thank you!

**Comment by mrapczynski** [comment]
@joelday 

Sure. List of installed extensions:
```
AdamCaviness.theme-monokai-dark-soda
DotJoshJohnson.xml
HookyQR.beautify
IBM.output-colorizer
PeterJausovec.vscode-docker
QassimFarid.ejs-language-support
TimonVS.ReactSnippetsStandard
bierner.color-info
christian-kohler.npm-intellisense
christian-kohler.path-intellisense
dbaeumer.vscode-eslint
dcortes92.vs-freemarker
donjayamanne.githistory
donjayamanne.python
eamodio.gitlens
fknop.vscode-npm
flesler.url-encode
formulahendry.auto-close-tag
formulahendry.auto-rename-tag
formulahendry.code-runner
idleberg.hopscotch
joelday.docthis
mitchdenny.ecdc
pprice.better-merge
redhat.java
robertohuertasm.vscode-icons
sandcastle.vscode-open
sandy081.todotasks
sleistner.vscode-fileutils
spywhere.guides
sysoev.language-stylus
taichi.react-beautify
vilicvane.es-quotes
wayou.vscode-todo-highlight
```

For the user config, these are the only things that I have alerted from the factory delivered settings:
```javascript
{
    "docthis.includeMemberOfOnClassMembers": false,
    "docthis.includeMemberOfOnInterfaceMembers": false,
    "editor.acceptSuggestionOnEnter": false,
    "editor.quickSuggestionsDelay": 250,
    "editor.rulers": [80],
    "window.zoomLevel": 0,
    "git.enableLongCommitWarning": false,
    "vsicons.dontShowNewVersionMessage": true
}
```

**Comment by joelday** [comment]
@mrapczynski Thanks. I'll try to get to this ASAP.

**Comment by mrapczynski** [comment]
@joelday 👍 

Also, my memory is foggy, but this may have started after either... 

(a) certain docthis update in the not too distant past
or (b) I did in fact install an extension that conflicts

When I first started using docthis, I don't recall the jumping behavior. I've been tolerating for it awhile now. My bad for not reporting it sooner.

**Comment by joelday** [comment]
@mrapczynski This should be fixed in 9.4.0.

**Comment by mrapczynski** [comment]
@joelday Tested, and so far working much better. Thank you!
